CONTRIBUTING.md

```diff
@@ -105,28 +109,71 @@ variables and push-button functionality.
 ```bash
-brew services start postgresql  # Or `service postgresql start` if you're on linux
+invoke postgresql
 ./manage.py runserver           # Alternatively, `python manage.py runserver`
 ```
-
-If you you want to run with local HTTPS (allowing webhook processing of videos), you'll need `runserver_plus` installed
-(should be in your virtual environment already if you ran `pipenv install --dev`) and a directory with cert files:
+Note that this bare-minimum setup __won't__ allow you to kick off any offline tasks (i.e., sending emails and building
+studies) from the UI. If you want to test offline jobs, you'll need to boot up RabbitMQ (needed for all offline tasks) 
+and Docker (needed for experiment builds only); finally starting the celery worker itself to consume messages from the
+message queue:
 ```bash
-./manage.py runserver_plus --nopin --cert-file ${PATH_TO_YOUR_CERTFILE}
+invoke docker
+rabbitmq-server                                                      # You may need `sudo` for this
+celery worker --app=project --loglevel=INFO -Q builds,email,cleanup
 ```
 
-If you want to hack on experiment building, you'll need to boot up RabbitMQ and Docker first, and finally the celery 
-worker itself:
+Serving with HTTPS will allow you to do two additional things:
+1. Process videos with the Pipe webhook
+2. Communicate with your locally served instance of ember-lookit-frameplayer (if you have configured it to serve in 
+HTTPS - not default, but recommended).
+
+To serve with HTTPS, you'll need `runserver_plus` installed (should be in your virtual environment already if you ran 
+`pipenv install --dev`) and a directory with cert files:
 ```bash
-open /Applications/Docker.app                                        # If you're on Linux, just `docker`
-rabbitmq-server                                                      # You may need `sudo` for this
-celery worker --app=project --loglevel=INFO -Q builds,email,cleanup
+./manage.py runserver_plus --nopin --cert-file ${PATH_TO_YOUR_CERTFILE}
 ```
 
-### Source Control
+### Outside Contributors
+If you are _not_ on the Lookit core team, this section is for you!
+
+#### Forking
+While core contributors will work directly on the original repo, outside contributors will need to maintain their own 
+fork of Lookit while working on new features for (or fixing bugs for) the platform. We recommend reading this handy
+[guide on forking][forking-workflow-walkthrough] if you are not familiar with this workflow. GitHub's own [`hub` 
+utility][hub-oss-contrib] will make your life a lot easier in this regard by abstracting over some of the steps; you 
+can find the installation instructions [here][hub-install] (or, if you like to live on the cutting-edge, try out 
+GitHub's [shiny new CLI][github-cli]).
+
+One additional note: when you submit a PR, you'll want to target the `develop` branch (if you forget, no biggie - we 
+can change that on the PR for you prior to merging any of your work).
```

Comment:
do we still want to encourage outside contributors to rebase on develop & name their branches based on issue number & type? or just not figure that at least for now we'll clean that sort of thing up for them?

Reply:
Nah - I think it's fine for outside contributors to do single-branch development if they're forking. Branch prefixing in git flow is mostly to coordinate against a single remote.